### PR TITLE
Add KlerosLiquidProxy tests

### PR DESCRIPTION
**Pull request by bbenligiray**
Closes https://github.com/api3dao/claims-manager/issues/63 by achieving full coverage
Closes https://github.com/api3dao/claims-manager/issues/64 though it only tests one case (enough imo)
Closes https://github.com/api3dao/claims-manager/issues/87

I also reordered some event fields to match ClaimsManager

### CODE REVIEW

File: contracts/arbitrators/KlerosLiquidProxy.sol [hunk 1]
```diff
@@ -116,7 +112,7 @@ contract KlerosLiquidProxy is Multicall, IKlerosLiquidProxy {
         string calldata evidence
     ) external payable override {
         bytes32 claimHash = keccak256(
-            abi.encodePacked(policyHash, msg.sender, claimAmountInUsd, evidence)
+            abi.encodePacked(policyHash, claimant, claimAmountInUsd, evidence)
```
Comment (bbenligiray) [inline contracts/arbitrators/KlerosLiquidProxy.sol:115]: :bug:

File: contracts/arbitrators/KlerosLiquidProxy.sol [hunk 2]
```diff
@@ -73,7 +69,7 @@ contract KlerosLiquidProxy is Multicall, IKlerosLiquidProxy {
             evidence: evidence
         });
         claimHashToDisputeIdPlusOne[claimHash] = disputeId + 1;
-        emit CreatedDispute(claimHash, claimant, disputeId);
+        emit CreatedDispute(claimant, disputeId, claimHash);
```
[the remark this answers is not in the document]
Comment (bbenligiray) [reply contracts/arbitrators/KlerosLiquidProxy.sol:72]: Normally we have indexed `claimant`, `policyHash` and `claimHash`. We definitely want `disputeId` to be indexed here, but EVM allows only up to 3 fields being indexed. Among these three, `policyHash` felt like the least important.

I considered
```
event CreatedDispute(
        address indexed claimant,
        bytes32 policyHash,
        bytes32 indexed claimHash,
        uint256 indexed disputeId
    );
```
for consistency but the main point of consistency if for indexing to work so I don't see much point in it. Also we don't even know `policyHash` while emitting `SubmittedEvidenceToKlerosArbitrator` and only have 3 fields to emit so it becomes awkward.

[the remark this answers is not in the document]
Comment (mcoetzee) [reply contracts/arbitrators/KlerosLiquidProxy.sol:72]: Ok cool fair enough 👍

**Review by mcoetzee — APPROVED** [review]

File: contracts/arbitrators/KlerosLiquidProxy.sol (hunk 1 above)
Comment (acenolaza) [reply contracts/arbitrators/KlerosLiquidProxy.sol:115]: 🐞

**Review by acenolaza — APPROVED** [review]
LGTM

**Pull request merged**